R/fixed_regex_linter.R

```diff
@@ -231,7 +235,7 @@ get_token_replacement <- function(token_content, token_type) {
       token_content
     }
   } else { # char_escape token
-    if (rex::re_matches(token_content, rex::rex("\\", one_of("^${}().*+?|[]\\<>:;")))) {
+    if (rex::re_matches(token_content, rex::rex("\\", one_of("^${}().*+?|[]\\<>:;/")))) {
```

Comment:
would you mind checking the ASCII table for any other common symbols that might be worth including?

Reply:
Good idea. I have expanded on the character set. I think this about covers all characters we might encounter in this context.